index.html

```diff
@@ -609,6 +625,106 @@ <h3>Ed25519Signature2018</h3>
           </pre>
       </section>
 
+<section id="Ed25519VerificationKey2020">
```

Comment:
nit: indendation

Reply:
yep, these are all over the place... imo the entire spect should be linted & formatted before each push.